docs/static/module-indexes/TerraformResourceModules.csv

```diff
@@ -8,6 +8,7 @@ Microsoft.Cdn,profiles,CDN Profile,avm-res-cdn-profile,New Module :new:,https://
 Microsoft.Compute,sshPublicKeys,Public SSH Key,avm-res-compute-sshpublickey,New Module :new:,https://github.com/Azure/terraform-azurerm-avm-res-compute-sshpublickey,https://registry.terraform.io/modules/Azure/avm-res-compute-sshpublickey/azurerm/latest,46d3xgtf.res.compute-sshpublickey,ChrisSidebotham,Chris Sidebotham,,,@Azure/avm-res-compute-sshpublickey-module-owners-tf,@Azure/avm-res-compute-sshpublickey-module-contributors-tf,AVM Resource Module for Public SSH Key,,
 Microsoft.Compute,virtualMachines,Virtual Machine,avm-res-compute-virtualmachine,Module Available :green_circle:,https://github.com/Azure/terraform-azurerm-avm-res-compute-virtualmachine,https://registry.terraform.io/modules/Azure/avm-res-compute-virtualmachine/azurerm/latest,46d3xgtf.res.compute-virtualmachine,jchancellor-ms,Jon Chancellor,,,@Azure/avm-res-compute-virtualmachine-module-owners-tf,@Azure/avm-res-compute-virtualmachine-module-contributors-tf,AVM Resource Module for Virtual Machine,TFVM location: https://github.com/Azure/terraform-azurerm-compute,2023-11
 Microsoft.Compute,virtualMachineScaleSets,Virtual Machine Scale Set,avm-res-compute-virtualmachinescaleset,New Module :new:,https://github.com/Azure/terraform-azurerm-avm-res-compute-virtualmachinescaleset,https://registry.terraform.io/modules/Azure/avm-res-compute-virtualmachinescaleset/azurerm/latest,46d3xgtf.res.compute-virtualmachinescaleset,terrymandin,Terry Mandin,marcelkmfst,Marcel Keller,@Azure/avm-res-compute-virtualmachinescaleset-module-owners-tf,@Azure/avm-res-compute-virtualmachinescaleset-module-contributors-tf,AVM Resource Module for Virtual Machine Scale Set,,
+Microsoft.CognitiveServices,accounts,Cognitive Account,avm-res-cognitiveservices-cognitive-account,New Module :new:,https://github.com/Azure/terraform-azurerm-avm-res-cognitiveservices-cognitive-account,https://registry.terraform.io/modules/Azure/avm-res-cognitiveservices-cognitive-account/azurerm/latest,46d3xgtf.res.cognitiveservices-cognitive-account,lonegunmanb,Zijie He,lonegunmanb,Zijie He,@Azure/avm-res-cognitiveservices-cognitive-account-module-owners-tf,@Azure/avm-res-cognitiveservices-cognitive-account-module-contributors-tf,AVM Resource Module for Cognitive Account,,2024-01
```

Comment:
The last value "2024-01" for column "FirstPublishedIn" is only provided when the module is finally published into the registry, recommend removing this value for now till such time the module gets published.